schemas/constructs/v1alpha1/capability.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$id": "https://schemas.meshery.io/capability.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Meshery manages entities in accordance with their specific capabilities. This field explicitly identifies those capabilities largely by what actions a given component supports; e.g. metric-scrape, sub-interface, and so on. This field is extensible. Entities may define a broad array of capabilities, which are in-turn dynamically interpretted by Meshery for full lifecycle management.",
+  "additionalProperties": false,
+  "type": "object",
+
+  "required": ["schemaVersion", "version", "displayName", "kind", "type", "state", "status"],
+
+  "properties": {
+    "schemaVersion": {
+      "$ref": "../core.json#/definitions/versionString",
+      "description": "Specifies the version of the schema to which the capability definition conforms."
+    },
+    "version": {
+      "$ref": "../core.json#/definitions/semverString",
+      "description": "Version of the capability definition.",
+      "minLength": 1,
```

Comment:
Shouldn't this be standard?
May be add in the semverString schema?

Reply:
makes sense :+1: